neqo-crypto/src/agentio.rs

```diff
@@ -321,8 +321,10 @@ unsafe extern "C" fn agent_available64(mut fd: PrFd) -> prio::PRInt64 {
     io.input.available as prio::PRInt64
 }
 
+#[allow(clippy::cast_possible_truncation)]
 unsafe extern "C" fn agent_getname(_fd: PrFd, addr: *mut prio::PRNetAddr) -> PrStatus {
     let a = addr.as_mut().unwrap();
+    // Cast is safe because prio::PR_AF_INET is 2
     a.inet.family = prio::PR_AF_INET as prio::PRUint16;
```

Comment:
Can the annotation be moved to the specific line?

Reply:
I think annotations inside blocks are experimental for now.

Edit: They are https://github.com/rust-lang/rust/issues/15701